doc/architecture/frontend.md

```diff
@@ -0,0 +1,57 @@
+# Frontend Architecture
+
+## Structure
+
+- `__mocks__/`: Contains mocked modules for tests
```

Comment:
Please describe why it has the special name: Is this used by some framework or is this some convention?